src/PKSim.Presentation/Presenters/TemplatePresenter.cs

```diff
@@ -121,6 +120,10 @@ private bool shouldLoadTemplateWithReferences(TemplateType templateType)
          if (!selectionSupportsReference(templateType))
             return false;
 
+         //For simulation subject always save linked expression profiles
+         if (templateType.Is(TemplateType.SimulationSubject))
```

Comment:
Not saving the expression profile will result in a corrupted individual. It needs to be saved